src/mlpack/core/optimizers/sgd/update_policies/momentum_update.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file momentum_update.hpp
+ * @author Arun Reddy
+ *
+ * Momentum update for SGD
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SGD_MOMENTUM_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SGD_MOMENTUM_UPDATE_HPP
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Momentum update policy for SGD.
+ *
+ */
```

Comment:
@rcurtin: Can I move the comments(as shown below) related to vanilla update in sgd.hpp to vanilla_update.hpp ? 

``` c++
Stochastic gradient descent iterates
 * over each function \f$ f_i(A) \f$, producing the following update scheme:
 *
 * \f[
 * A_{j + 1} = A_j + \alpha \nabla f_i(A)
 * \f]
 *
 * where \f$ \alpha \f$ is a parameter which specifies the step size.  \f$ i \f$
 * is chosen according to \f$ j \f$ (the iteration number).

```